idl/chromadb/proto/coordinator.proto

```diff
@@ -64,7 +64,8 @@ message CreateTenantResponse {
 }
 
 message GetTenantRequest {
-  string name = 1;
+  optional string name = 1;
+  optional string resource_name = 2;
```

Comment:
Calling what is the `id` of the tenant "name" is an unfortunate hangover of the past (unless there is historical context I'm missing), but leaving it be for now. 

Adding `resource_name` here so that once these changes are deployed we're able to update the client/server to query for tenants by `resource_name`.